ohmec_data_na.js

```diff
@@ -4,6 +4,146 @@ dataNA = {
   "_comment3":"SPDX-License-Identifier: Apache-2.0",
   "type":"FeatureCollection",
   "features":[
+    { "type":"Feature",
+      "id":"OH700",
+      "properties":{
+        "entity1type":"nation",
+        "entity1name":"Native American",
+        "entity2type":"people",
+        "entity2name":"Ohlone",
+        "labelRotate":45,
+        "labelY":0,
+        "labelX":0,
+        "labelScale":2.0,
+        "fidelity":2,
+        "editdatestr":"2021:09:10",
+        "source":"https://en.wikipedia.org/wiki/Ohlone",
+        "startdatestr":"700:01:01",
```

Comment:
I've indicated in format descriptor a new notion to just put "year only" for the date to indicate the amount of accuracy conveyed. So these should be `startdatestr` = `"700"` and `enddatestr` = `"1768"` (end dates snap to december 31st).